From 914ad7b2cd53b8396a6e407c91b63adfc3b18443 Mon Sep 17 00:00:00 2001
From: orbea <orbea@riseup.net>
Date: Wed, 8 Jan 2020 07:42:21 -0800
Subject: [PATCH] Makefile: Install the text directory.

This fixes a crash at the end of the tutorial level.

antares-glfw: text/6000.txt: couldn't find resource "text/6000.txt"

Signed-off-by: orbea <orbea@riseup.net>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 7146731e..7404fb34 100644
--- a/Makefile
+++ b/Makefile
@@ -105,6 +105,7 @@ install-data: build
 	cp -r data/sounds $(DESTDIR)$(DATADIR)/app
 	cp -r data/sprites $(DESTDIR)$(DATADIR)/app
 	cp -r data/strings $(DESTDIR)$(DATADIR)/app
+	cp -r data/text $(DESTDIR)$(DATADIR)/app
 
 .PHONY: install-scenario
 install-scenario: build
